src/app/project/documents/documents-tab.component.ts

```diff
@@ -455,7 +455,7 @@ export class DocumentsTabComponent implements OnInit, OnDestroy {
       pageNumber,
       this.tableParams.pageSize,
       this.tableParams.sortBy,
-      { documentSource: 'PROJECT' },
+      { documentSource: 'PROJECT', eaoStatus: 'Published' },
```

Comment:
I am not 100% sure, but I believe this was removed as it was deemed to cause a bug.

Reply:
good catch. This was probably from a bad rebase on my end.